src/routes/boot.ts

```diff
@@ -210,6 +211,7 @@ const loggedInBoot = async (
     flags: adjustFlagsToUser(flags, user),
     alerts: {
       ...excludeProperties(alerts, ['userId']),
```

Comment:
Should we exclude also the `lastChangelog` property here?

Reply:
@idoshamun we decided that we do need `lastChangelog`, so we are leaving both. This is because updating boot is not really easy currently on the frontend (without invalidation, which is expensive). 

If we would leave only `changelog` we would need to have a computed property on graphql read/update alerts query which is not the best pattern we think - since `changelog` currently only exists in boot. 

So to sumarize we use:
- we use `changelog` to quickly know on webapp startup if we need to fetch the latest changelog article
- we then use `lastChangelog` for updating and hiding it after the update

FYI @rebelchris